gulpfile.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+// mostly copy-pasta from https://github.com/jsperf/benchmark.js-wrapper/blob/c84d86d3a6a15a94a1acb2b99ec24562adfa1d27/gulpfile.js
+
+const gulp = require('gulp');
+
+const addSrc = require('gulp-add-src');
+const concat = require('gulp-concat');
+const insert = require('gulp-insert');
+const remoteSrc = require('gulp-remote-src');
+const replace = require('gulp-replace');
+// const uglify = require('gulp-uglify');
+
+const BENCHMARKJS_VERSION = require('./package.json').devDependencies.benchmark;
+
+gulp.task('js', function () {
+  /*
+    Put together in order:
+    1. lodash
+    2. platform
+    3. benchmark
+    4. ui.js
+    5. ui.browserscope.js
+  */
+  remoteSrc([
+    `bestiejs/benchmark.js/${BENCHMARKJS_VERSION}/plugin/ui.browserscope.js`
+  ], {
+    base: 'https://raw.githubusercontent.com/',
+    requestOptions: {
+      gzip: true,
+      strictSSL: true
+    }
+  })
+
+	// list in reverse order since prepending
+  // use `require.resolve` so you get file path and not file contents
+  .pipe(addSrc.prepend('client/ui.js'))
+  .pipe(addSrc.prepend(require.resolve('benchmark')))
+  .pipe(addSrc.prepend(require.resolve('platform')))
+  // Use whatever version of lodash Benchmark.js is using (hopefully?)
```

Comment:
I _think_ this is going to resolve to lodash@3.10.x from another dependency and **not** lodash@4 [like benchmark wants](https://github.com/bestiejs/benchmark.js/blob/c10dd81f2e350f0c0cb2d1dc2f59a5e1bb7cf0f3/package.json#L21). is that ok? should we add lodash@4 as a devDep? is there a way to reference `benchmark`'s dependencies?